pkgs/development/python-modules/vmprof/default.nix

```diff
@@ -21,6 +22,23 @@ buildPythonPackage rec {
   buildInputs = [ libunwind ];
   propagatedBuildInputs = [ colorama requests six pytz ];
 
+  patches = [
+    (fetchpatch {
+      name = "${pname}-python-3.10-compat.patch";
+      url = "https://github.com/vmprof/vmprof-python/pull/198.patch";
```

Comment:
For merged PRs, please use the commits that actually end up in an upstream branch. The PRs (or the commits behind them) can in theory disappear. In this case the commit would be https://github.com/vmprof/vmprof-python/commit/e4e99e5aa677f96d1970d88c8a439f995f429f85